### DISCUSSION TITLE
Device-Availability vs last_seen - Which one to use ?

### DISCUSSION THREAD

**Discussion by gelsas** [Koenkk/zigbee2mqtt#12263] [General]
Hello everybody, 

Which of these should preferably be used to log/check if devices are online/responsive/connected ?

Device-Availability 
https://www.zigbee2mqtt.io/guide/configuration/device-availability.html

or 
last_seen 
https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request

**Comment by Koenkk** [comment]
Device availability
